README.md

```diff
@@ -60,8 +60,12 @@ A list of of AI coding topics.
 
 ## Products & Startups
 
-- [Copilot](https://github.com/features/copilot) + [Copilot X](https://github.com/features/preview/copilot-x)
+### Test Case Generation
 - [Codeium](https://www.codeium.com/)
```

Comment:
Codeium is not for test case generation, guessing you mean codium ?

Reply:
Yes.